RHEL/7/templates/static/bash/ensure_gpgcheck_repo_metadata.sh

```diff
@@ -1,8 +1,15 @@
 # platform = Red Hat Enterprise Linux 7
 
 if grep --silent ^repo_gpgcheck /etc/yum.conf ; then
-        sed -i "s/^repo_gpgcheck.*/repo_gpgcheck=1/g" /etc/yum.conf
-else
-        echo -e "\n# Set repo_gpgcheck to 1 per security requirements" >> /etc/yum.conf
-        echo "repo_gpgcheck=1" >> /etc/yum.conf
+        sed -i "s/^repo_gpgcheck.*//g" /etc/yum.conf
+fi
+if ! grep --silent "# repo_gpgcheck=1" /etc/yum.conf ; then
+        cat << EOF >> /etc/yum.conf
+# Set repo_gpgcheck to 1 per security requirements.
+# Warning: This setting setting is temporarily commented out,
+# because it disables installation of packages from repositories
+# that do not provide metadata signatures, including RHEL repositories.
+# repo_gpgcheck=1
+EOF
```

Comment:
Sure. We can always easily add it back in. Don't remember if there is an ansible file for this or not.